wordpress-mobile/WordPress-Android
# Add track events for quick link ribbon

Fixes #16272 

This PR adds the support for tracking events when tapping the quick link ribbon buttons. Pages/Posts/Media/Stats

Note: Only 1 approval is needed. Thanks

**To test:**
- Launch the app
- Navigate to Me -> App Settings -> Debug Settings
- Enable MySiteDashboardTabsFeatureConfig and restart the app
- Navigate to My Site View
- Tap on the following (returning back to My Site View after each click):  
   - All Quick Link buttons (Stats, Pages, Posts, and Media) 
-  Swipe down on the My Site View to request a pull-to-refresh
- View the logs and note that "tab_source=dashboard" is included in the properties of the track event.  
```
🔵 Tracked: quick_action_ribbon_tapped, Properties: {"default_tab_experiment":"nonexistent","tab_source":"dashboard","button":"pages"}
🔵 Tracked: quick_action_ribbon_tapped, Properties: {"default_tab_experiment":"nonexistent","tab_source":"dashboard","button":"posts"}
🔵 Tracked: quick_action_ribbon_tapped, Properties: {"default_tab_experiment":"nonexistent","tab_source":"dashboard","button":"media"}
🔵 Tracked: quick_action_ribbon_tapped, Properties: {"default_tab_experiment":"nonexistent","tab_source":"dashboard","button":"stats"}
```

## Regression Notes
1. Potential unintended areas of impact
Events for quick link ribbon buttons are not tracked. 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests and Unit tests  

3. What automated tests I added (or what prevented me from doing so)
Added tests in `MySiteViewModelTest.kt`

PR submission checklist:
